### PR TITLE
feat(tabs): add opportunity to add multiple classes to customClass attribute

### LINKED ISSUE #2467: Tab: customClass doesn allow multiply class
Hi, 
My code is .... [customClass]="getStyles()"
getStyles returns "hide" or "hide invalid". In case "hide invalid" I get error "The token provided ("hide invalid") contains HTML space characters, which are not valid in tokens." because of splitting classes by space.
This is because in beta 4, for customClass setter you use this.renderer.setElementClass which does not allow multiply classes separated by space.
How can i continues using multiply classes for customClass?

### PR DESCRIPTION
**Pull request by LimarenkoDenis**
Fixes #2467

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [ ] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

### CODE REVIEW

**Comment by IlyaSurmay** [comment]
Hi, thanks for PR!
We can't merge it because looks like it hasn't been tested. This is what I get on tab init stage

Please, fix your PR, test it and update, then we'll merge it.

File: src/tabs/tab.directive.ts [hunk 1]
```diff
@@ -29,21 +29,15 @@ export class TabDirective implements OnInit, OnDestroy {
...
   set customClass(customClass: string) {
-    if (this._customClass && this._customClass !== customClass) {
-      this.renderer.removeClass(
-        this.elementRef.nativeElement,
-        this._customClass
-      );
-    }
-
     this._customClass = customClass;
+    this._customClass.split(' ').forEach((element: string) => this.setElementClass = element);
```
Comment (IlyaSurmay) [inline src/tabs/tab.directive.ts]: this will throw an error if customClass isn't set

Comment (IlyaSurmay) [inline src/tabs/tab.directive.ts]: this will throw an error if customClass isn't set

File: src/tabs/tab.directive.ts [hunk 2]
```diff
@@ -29,21 +29,15 @@ export class TabDirective implements OnInit, OnDestroy {
...
+  }
 
-    if (this._customClass) {
-      this.renderer.addClass(
-        this.elementRef.nativeElement,
-        this._customClass
-      );
-    }
+  set setElementClass(customClass: string) {
+    this.renderer.addClass(
```
Comment (IlyaSurmay) [inline src/tabs/tab.directive.ts]: what if list of classes changes? for example, if user pass `class-1 class-2` and then `class-2 class-3`. in this case all 3 classes will be appended

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Comment by valorkin** [comment]
Thanks for PR :)

**Pull request merged**